surefire-providers/surefire-junit-platform/src/test/java/org/apache/maven/surefire/junitplatform/JUnitPlatformProviderTest.java

```diff
@@ -220,6 +228,13 @@ public void rerunStillFailing()
         assertEquals( 0, summary.getTestsSucceededCount() );
         assertEquals( 0, summary.getTestsAbortedCount() );
         assertEquals( 3, summary.getTestsFailedCount() );
+        failDisplays.clear();
+        for ( TestExecutionSummary.Failure failure : summary.getFailures() )
```

Comment:
Pls use brackets. We will use the checkstyle on tests which would fail the build if you do not type brackets. Thx